src/DeterministicVIImagePSF.jl

```diff
@@ -49,17 +49,18 @@ function infer_source_fft(images::Vector{Image},
                           neighbors::Vector{CatalogEntry},
                           entry::CatalogEntry;
                           min_radius_pix=Nullable{Float64}())
-   cat_local = vcat([entry], neighbors)
-   vp = init_sources([1], cat_local)
-   patches = get_sky_patches(images, cat_local)
-   load_active_pixels!(images, patches, min_radius_pix=min_radius_pix)
-
-   ea_fft, fsm_mat = initialize_fft_elbo_parameters(
-       images, vp, patches, [1], use_raw_psf=true)
-   elbo_fft_opt = get_fft_elbo_function(ea_fft, fsm_mat)
-   maximize_f(elbo_fft_opt, ea_fft, max_iters=150)
-
-   vp[1]
+    cat_local = vcat([entry], neighbors)
+    vp = init_sources([1], cat_local)
+    patches = get_sky_patches(images, cat_local)
+    load_active_pixels!(images, patches, min_radius_pix=min_radius_pix)
+
+    ea_fft, fsm_mat = initialize_fft_elbo_parameters(images, vp, patches, [1], use_raw_psf=true)
+    elbo_fft_opt = get_fft_elbo_function(ea_fft, fsm_mat)
+    bound_params = ea.vp[ea.active_sources]
```

Comment:
@jrevels This line should be ```bound_params = ea_fft.vp[ea_fft.active_sources]``` presumably. Not sure if that's the source of the test error you're seeing.